recipes/biobox/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "biobox" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Degiacomi-Lab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 749f5a546b4c6e17c369bf487c7ef08e5970f9da6c2f1b790a8e6efbb4804e85
+
+build:
+  number: 0
+  script:
+        {{ PYTHON }} -m pip install . -vv
```

I'm sorry. I should've probably been more specific here. You'll need:
```suggestion
  script:
      - {{ PYTHON }} -m pip install . -vv
```